source/stylesheets/_variables.scss

```diff
@@ -28,18 +28,18 @@ $examples-bg: #393939 !default;
 $code-bg: #292929 !default;
 $code-annotation-bg: #191D1F !default;
 $nav-subitem-bg: #f7f7f7 !default;
-$nav-active-bg: #9B5C8F !default;
+$nav-active-bg: #7f54b3 !default;
 $nav-active-parent-bg: #f7f7f7 !default; // parent links of the current section
 $lang-select-border: #000 !default;
 $lang-select-bg: #222 !default;
 $lang-select-active-bg: $examples-bg !default; // feel free to change this to blue or something
 $lang-select-pressed-bg: #111 !default; // color of language tab bg when mouse is pressed
 $main-bg: #ffffff !default;
-$aside-notice-bg: #9B5C8F !default;
+$aside-notice-bg: #7f54b3 !default;
 $aside-warning-bg: #ca4949 !default;
 $aside-success-bg: #38a845 !default;
 $search-notice-bg: #c97a7e !default;
-$link-color: #804877;
```

Comment:
We were previously using a different purple here than the bg colors. Do you know if this should be the case here as well?

Reply:
@jonathansadowski good question. I noticed that it was quite similar, and I wasn't sure the reason for having two similar colours, so I used our Woo purple for both here. 

old link color: ![IMG002397](https://user-images.githubusercontent.com/1509380/236050647-3c7db4ff-11de-4512-ab3d-3ca8ea0a2914.png) 
new link color: ![IMG002399](https://user-images.githubusercontent.com/1509380/236051019-0e764e42-fec0-48c7-8965-46d36de27356.png)


old bg color: ![IMG002398](https://user-images.githubusercontent.com/1509380/236050714-636f1f1d-10ba-4c3a-a9f6-5f19921ab9fa.png) 
new bg color:![IMG002399](https://user-images.githubusercontent.com/1509380/236051029-8f6dac82-9d23-4e61-9810-0180cca9f11f.png)

Reply:
Okay, yep, that seems reasonable to me. I'll merge this, thanks again!